### ISSUE TITLE
Logo not stored

### ISSUE THREAD

**Issue by Tychodewaard** [dnnsoftware/Dnn.Platform#4272]
## Description of bug
If you go to site settings and upload a logo, it is not stored. If you then select the logo by browse to file (the logo is uploaded in the previous step just not stored as a setting) the logo disappears from the website

## Steps to reproduce
See above

## Expected behavior
Logo works like before. You can upload a logo or browse through the files. In both scenarios, the logo is displayed in the site

In the admin log I see
Message:Job #07838251-35b7-43d9-9ced-860761fc267d (Item dnnfile-80 from portal 0) is skipped and will be removed from queue. Job details: {"Guid":"07838251-35b7-43d9-9ced-860761fc267d","Priority":5,"Due":"2020-11-03T09:50:05.0633333+00:00","Action":"add update","BehaviorId":1,"PortalId":0,"TabId":-1,"ModuleId":-1,"ItemId":"dnnfile-80","Contents":null,"ContentType":"application/octet-stream","Metadata":{"PortalId":0,"ItemId":null,"Title":"","LockTitle":false,"OriginalId":null,"OriginalName":null,"Origina

## Affected version
* [x] 09.08.00 release candidate

**Comment by valadas** [comment]
I cannot reproduce the issue on a clean 9.8.0 RC1 install or in the current "nightly build" on the release branch. Is there other setup steps done in order to make this behavior happen? What format is the logo? Any other clues ?

**Comment by Tychodewaard** [comment]
Ok...it took some perseverance as I usually don't have this scenario.
The first logo I uploaded was a SVG. This is not working in DNN/Xcillion. So, I uploaded a jpg. As nothing changed, I think I was editing the wrong language...(if you switch language, the settings change but the url stays the same, so I guess I wasn't paying attention).
Then I browsed to the file but I did not hit 'enter'. I just discovered that this is necessary instead of 'save' being enough.

So, as I did all of this: I have a logo!
